code/game/objects/effects/anomalies.dm

```diff
@@ -378,6 +378,9 @@
 	for(var/turf/T in oview(get_turf(src), 7))
 		turf_targets += T
 
+	for(var/mob/living/carbon/human/H in view(get_turf(src), 3))
```

Comment:
Why only humans? Why not carbons in general?

Reply:
It could be generalized to carbons, but I'm pretty sure that xenos aren't affected by cold, and it'd be rare for them to interact with an anomaly anyways

Reply:
It looks their body temperature does get impacted, and in fact `bodytemperature` is a mob-level trait. On that note, you can probably even just shoot at any `mob/living` in sight here.

Reply:
Alright, I'll do that in the next commit